components/composition/GridItem/GridItem.stories.ts

```diff
@@ -0,0 +1,81 @@
+import { StoryObj } from '@storybook/web-components';
```

Comment:
Wouldn't it make the dependency clearer if grid item lived in the grid folder? I can't think of any case where grid item would be used outside grid.

Reply:
I think we could still keep a separate storyfile using the storyname to put it under the grid folder in the storybook sidebar